File: app/src/main/java/br/com/patrocine/patrocine/model/Movie.java
package br.com.patrocine.patrocine.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.ArrayList;

public class Movie implements Serializable {

    @SerializedName("id")
    private int id;
    @SerializedName("title")
    private String title;
    @SerializedName("full_title")
    private String fullTitle;
    @SerializedName("sinopse")
    private String sinopse;
    @SerializedName("duration")
    private int duration;
    @SerializedName("genre")
    private String genre;
    @SerializedName("censorship")
    private String censorship;
    @SerializedName("image")
    private String image;
    @SerializedName("image_mini")
    private String image_mini;
    @SerializedName("slide01")
    private String slide01;
    @SerializedName("slide02")
    private String slide02;
    @SerializedName("slide03")
    private String slide03;
    @SerializedName("trailer")
    private String trailer;
    @SerializedName("sessions")
    private ArrayList<Grid> grid;

    public Movie(){
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getFullTitle() {
        return fullTitle;
    }

    public void setFullTitle(String fullTitle) {
        this.fullTitle = fullTitle;
    }

    public String getSinopse() {
        return sinopse;
    }

    public void setSinopse(String sinopse) {
        this.sinopse = sinopse;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public String getCensorship() {
        return censorship;
    }

    public void setCensorship(String censorship) {
        this.censorship = censorship;
    }

    public String getImage() {
        this.image = "https://api.patrocine.com.br/v1/static/images/covers/" + this.image;
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getImage_mini() {
        return image_mini;
    }

    public void setImage_mini(String image_mini) {
        this.image_mini = image_mini;
    }

    public String getSlide01() {
        return slide01;
    }

    public void setSlide01(String slide01) {
        this.slide01 = slide01;
    }

    public String getSlide02() {
        return slide02;
    }

    public void setSlide02(String slide02) {
        this.slide02 = slide02;
    }

    public String getSlide03() {
        return slide03;
    }

    public void setSlide03(String slide03) {
        this.slide03 = slide03;
    }

    public String getTrailer() {
        return trailer;
    }

    public void setTrailer(String trailer) {
        this.trailer = trailer;
    }

    public ArrayList<Grid> getGrid() {
        return grid;
    }

    public void setGrid(ArrayList<Grid> grid) {
        this.grid = grid;
    }
}

File: app/src/main/java/br/com/patrocine/patrocine/model/Movie.kt
package br.com.patrocine.patrocine.model

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.util.ArrayList

class Movie: Serializable {

    @SerializedName("id")
    var id: Int = 0
    @SerializedName("title")
    var title: String? = null
    @SerializedName("full_title")
    var fullTitle: String? = null
    @SerializedName("sinopse")
    var sinopse: String? = null
    @SerializedName("duration")
    var duration: Int = 0
    @SerializedName("genre")
    var genre: String? = null
    @SerializedName("censorship")
    var censorship: String? = null
    @SerializedName("image")
    var image: String? = null
    @SerializedName("image_mini")
    var image_mini: String? = null
    @SerializedName("slide01")
    var slide01: String? = null
    @SerializedName("slide02")
    var slide02: String? = null
    @SerializedName("slide03")
    var slide03: String? = null
    @SerializedName("trailer")
    var trailer: String? = null
    @SerializedName("sessions")
    var grid: ArrayList<Grid>? = null

}